lib/hexpm_web/controllers/api/package_controller.ex

```diff
@@ -44,6 +44,17 @@ defmodule HexpmWeb.API.PackageController do
     end
   end
 
+  def audit_logs(conn, params) do
+    if package = conn.assigns.package do
+      audit_logs = AuditLogs.all_by(package, Hexpm.Utils.safe_int(params["page"]))
+
+      conn
+      |> render(:audit_logs, audit_logs: audit_logs)
+    else
+      not_found(conn)
```

Comment:
This logic is from `show/2`. Shall we move this `not_found` handling logic to `maybe_fetch_package`?

Reply:
No, I think this simple `if` is fine to duplicate.